src/pages/search/index.js

```diff
@@ -79,6 +80,23 @@ export const Search = ({
     }
   }, [])
 
+  useEffect(() => {
+    const {
+      downloadable_organism: organism,
+      platform,
+      technology,
+      search
+    } = query
+
+    if (search) {
+      gtag.trackSearchTerm(userSearchTerm)
+    }
+
+    if (!!organism || !!platform || !!technology) {
+      gtag.trackFilterCombination(query)
+    }
```

Comment:
```suggestion
    gtag.trackFilterCombination(query)
```

We can wrap the two tracking calls here into one exposed `trackSearchQuery` and then conditionally track the different parts under the analytics hood.

Reply:
I've renamed the methods called from `trackSearchQuery` to `trackSearchTermFromQuery` and `trackSearchTermFromQuery` respectively (pls see at f7159492683e61bc53cc65cfa75bc412a270c606).